src/routes/docs/advanced/platform/billing/+page.markdoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: article
+title: Billing
+description: Understand Appwrite's billing features, like budget caps, billing periods, taxes, and more.
+---
+
+Appwrite allows you to configure billing per organization. 
+You can access your organizations billing information under the **Billing** tab of your organization.
+
+# Plans {% #plans %}
+You can view or change your organization's plan under the **Plan summary** section. 
+You'll also find the expected cost, as well as the start and end date of the current billing period.
+
+## Billing period {% #billing-period %}
+Billing periods begin the day you change your plan, and lasts 30 days.
+You resource limits are reset at the beginning of each billing period and you will be billed for your plan and usage on the last day of each billing period.
+
+# Payment history {% #payment-history %}
+You can view and download you past payment's invoices under **Payment history**. 
+You can click the three-dots menu to view and download your invoices.
+
+# Payment methods {% #payment-methods %}
+Appwrite Cloud accepts credit card as payment methods and will bill the card at the end of each billing cycle.
+Appwrite accepts Visa, Mastercard, American Express, Discover & Diners Club, China UnionPay, Japan Credit Bureau (JCB), Cartes Bancairies, and eftpos.
+
+# Billing addresss {% #billing-addresss %}
+Your billing address will be displayed on your invoices and used when Appwrite Cloud bills your payment method at the end of a billing cycle.
+
+# Tax ID {% #tax-id %}
+If you'd like you or your company's tax ID displayed on your generated invoice, you can provide it under **Tax ID**.
+
+# Budget cap {% #budget-cap %}
```

Comment:
I think we should also talk about billing alerts here in a separate section

Reply:
When do we send alerts and can they be configured?

I'll add a whole new section about all the alerts.